maintenance/recipes/turn_off.rb

```diff
@@ -0,0 +1,3 @@
+bash 'turn maintenance off' do
+  code 'sudo -S -u deploy rm /srv/www/jiffyshirts/current/public/offline'
+end
```

Comment:
Also there's a chef resource `file` for that purpose:
```suggestion
file '/srv/www/jiffyshirts/current/public/offline' do
  action : delete
end
```

Reply:
Updated